phpunit.xml

```diff
@@ -8,6 +8,7 @@
     <testsuite name="Unit">
       <directory suffix="Test.php">./tests/Feature</directory>
       <directory suffix="Test.php">./tests/Unit</directory>
+      <directory suffix="Test.php">./tests/PHPUnit</directory>
```

Comment:
Please don't create another testsuite. Those tests should be placed under `Unit`

Reply:
@asbiin, Unit tests in Laravel are not a classic pure unit test, because they bootstrap the kernel, database, and much more. A pure unit test only works with abstractions and does not depend on anything, for this reason it is cleaner and faster. I can move the tests to the unit folder, but I would still suggest separating test types

Reply:
@asbiin, change reverted